### PR TITLE
docs: provide option for installing using a specific version via Helm

### PR DESCRIPTION
**Pull request by david-yu**
### Description
Provide an option in default install that describes how to install with a specific version of the Helm chart. 

### Testing & Reproduction steps
* In the case of bugs, describe how to replicate
* If any manual tests were done, document the steps and the conditions to replicate
* Call out any important/ relevant unit tests, e2e tests or integration tests you have added or are adding

### Links
Include any links here that might be helpful for people reviewing your PR (Tickets, GH issues, API docs, external benchmarks, tools docs, etc). If there are none, feel free to delete this section.

Please be mindful not to leak any customer or confidential information. HashiCorp employees may want to use our internal URL shortener to obfuscate links.

### PR Checklist

* [ ] updated test coverage
* [ ] external facing docs updated
* [ ] not a security concern
* [ ] checklist [folder](./../docs/config) consulted

### CODE REVIEW

File: website/content/docs/k8s/installation/install.mdx [hunk 1]
```diff
@@ -156,17 +156,20 @@ The Consul Helm only supports Helm 3.2+. Install the latest version of the Helm
...
-The Helm chart does everything to set up a recommended Consul-on-Kubernetes deployment.
-After installation, a Consul cluster will be formed, a leader will be elected, and every node will have a running Consul agent.
+1. Install Consul on Kubernetes using Helm. The Helm chart does everything to set up a recommended Consul-on-Kubernetes deployment. After installation, a Consul cluster will be formed, a leader will be elected, and every node will have a running Consul agent. 
+    1. To install the latest version of Consul on Kubernetes, issue the following command to install Consul with the default configuration using Helm. You could also install Consul on a dedicated namespace of your choosing by modifying the value of the `-n` flag for the Helm install.
+
+       ```shell-session
+       $ helm install consul hashicorp/consul --set global.name=consul --create-namespace --namespace consul
+       ```
+   
+    1. To install a specific version of Consul on Kubernetes, issue the following command which utilizes the `--version` flag to install with the default configuration using Helm.
```
Comment (im2nguyen) [inline website/content/docs/k8s/installation/install.mdx:166]:
```suggestion
    1. To install a specific version of Consul on Kubernetes, run the following command with `--version` flag to install the specified version with the default configuration using Helm.  
```

**Review by im2nguyen — APPROVED** [review]

File: website/content/docs/k8s/installation/install.mdx (hunk 1 above)
Comment (david-yu) [reply website/content/docs/k8s/installation/install.mdx:166]: @im2nguyen the `issue` verb was suggested by @trujillo-adam so perhaps we should leave it for now?

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/667128.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 19d91de896fe81229cc04468a5f735bf05a54e7b onto `release/1.12.x` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 19d91de896fe81229cc04468a5f735bf05a54e7b onto `release/1.11.x` succeeded!

**Pull request merged**